pkg/emb6/patches/0004-Provide-event-type-for-RIOT-s-conn_.-_send.patch

```diff
@@ -23,7 +23,7 @@ index 5316a67..78153be 100644
  #define EVENT_TYPE_ICMP6            0x05    ///< New icmp6 packet event
  #define EVENT_TYPE_TCPIP            0x06    ///< New tcpip event
  #define EVENT_TYPE_SLIP_POLL        0x07    ///< Process slip handler
-+#define EVENT_TYPE_CONN_SEND        0x08    ///< Send command from conn layer
++#define EVENT_TYPE_CONN_SEND        0x08    ///< Send command from sock layer
```

Comment:
wouldn't `EVENT_TYPE_SOCK_SEND` be more appropriate, though it requires to change every usage of this macro, too.